core-bundle/tests/Functional/RoutingTest.php

```diff
@@ -944,4 +1031,11 @@ public function testOrdersThePageModelsByCandidates(): void
         $this->assertSame(200, $response->getStatusCode());
         $this->assertStringContainsString('', $title);
     }
+
+    private function loadFixtureFiles(array $fileNames)
```

Comment:
Since this is a private method and you are always passing `theme` as first array value, why not add `theme` here by default?

Reply:
We could… but I tried to make it more explicit what's actually loaded…